README.md

```diff
@@ -1,237 +1,290 @@
 # [Contribute To This Project](https://syknapse.github.io/Contribute-To-This-Project/)
 
-![image info](/favicon.png)
+![image info](favicon.png)
 
-[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Contribute%20To%20This%20Project.%20An%20easy%20project%20for%20first-time%20contributors,%20with%20a%20full%20tutorial.%20By%20@Syknapse&url=https://github.com/Syknapse/Contribute-To-This-Project&hashtags=100DaysofCode 'Tweet this project')
+> Logo Created with :sparkling_heart: By [CandidDeer](https://github.com/CandidDeer)
 
+[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
```

Comment:
I like the use of the reference `twit`, good improvement. But please keep the twitter button on it's own line